app/src/main/java/com/zulip/android/MessageListFragment.java

```diff
@@ -53,6 +54,9 @@
 
         void openCompose(final MessageType type, String stream, String topic,
                          String pmRecipients);
+        void addToList(Message message);
+        void unMuteTopic(int StreamID,String subject);
+        void MuteTopic(int StreamID,String subject);
```

Comment:
please use standard Java conventions for function names here. 

> Compose method names using mixed case letters, beginning with a lower case letter and starting each subsequent word with an upper case letter.

source: https://www.cwu.edu/~gellenbe/javastyle/method.html